server/index.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express');
+const path = require('path');
+const {
+  sequelize: {
+    models: {
+      Product, Shop, Review, Reviewer, ReviewPhoto,
+    },
+  },
+} = require('../db');
+
+const app = express();
+const PORT = 4000;
+
+// Access components in build.js
+app.use(express.static(path.join(__dirname, '/../public/dist')));
+
+app.use('/listing/:id', (req, res) => {
+  let apiData = {
+    reviewPhotos: [],
+  };
+
+  Product.findByPk(req.params.id, {
+    include: [{
+      model: Review,
+      as: 'productReviews',
+      include: [{ model: Reviewer }, { model: ReviewPhoto }],
+    },
+    { model: Shop }],
+  })
+    .then((data) => {
+      const {
+        dataValues: {
+        // eslint-disable-next-line no-shadow
+          Reviewer, productReviews, Shop, ...productData
+        },
+      } = data;
+      apiData = { productData, shopData: Shop.dataValues };
```

Comment:
This line is getting rid of the array you initially defined in apiData at the top of the file. Perhaps change the above declaration to be an empty object to avoid confusion. Or maybe change this to decorating apiData if that suits better.